src/language-service/src/schemas/ui-lovelace.ts

```diff
@@ -552,6 +553,16 @@ export interface CastEntityConfig {
   hide_if_unavailable?: boolean;
 }
 
+export interface ButtonEntityConfig {
```

Comment:
Missing options:
- `entity`
- `icon_height`
- `show_name`
- `show_icon`
- `show_state`
- `state_color`
- `theme`

Reply:
Maybe I missinterpreted the code, but this is not the same as the normal button-entity, its the "special row button" under the Entities-card.

Which means in my mind that the schema needs to match the code in the link above: https://github.com/home-assistant/frontend/blob/96110637d935adc1c6278cc50319a6abc43eeb67/src/panels/lovelace/special-rows/hui-button-row.ts

From what I can gather there, I am missing `entity`, but the other options you refer to is not valid, do you agree?

(if you agree, I will also add `entity` to the documentation here, as optional: https://www.home-assistant.io/lovelace/entities/#button)